gulp_tasks/replace.js

```diff
@@ -7,16 +7,17 @@ module.exports = {
 		return gulp.src( [
 			`${ pkg._core_theme_pcss_path }base/_icons.pcss`,
 		] )
-			.pipe( replace( /url\('fonts\//g, 'url(\'var(--path-fonts)/icons-core/' ) )
+			.pipe( replace( /url\('fonts\/(.+)'\) /g, 'url(var(--path-fonts)/icons-core/$1) ' ) )
```

Comment:
This was necessary b/c the latest versions of postcss-custom-properties doesn't appear to be replacing properties within quoted strings, e.g. - `url('var(--path-fonts)/icons-core/')` doesn't get replaced with `url('/wp-content/themes/core/fonts/icons-core/')` when the task runs. Removing the quotes fixes it. And, per the CSS spec, quotes aren't required for `url()`'s.